core/src/rpc.rs

```diff
@@ -61,9 +61,59 @@ pub struct ReplyInfo {
     /// Value attached to the reply.
     pub value: u128,
     /// Reply code of the reply.
+    #[cfg_attr(feature = "std", serde(with = "serialize_reply_code"))]
```

Comment:
https://docs.rs/const-hex/latest/const_hex/serde/index.html
const-hex could also be used here. this dependency also exists in alloy.